.github/workflows/tests.yml

```diff
@@ -19,13 +19,22 @@ jobs:
       - name: Start Docker container with localstack
         if: matrix.session == 'localstack'
         run:  docker run -d -p 4566:9324 --rm softwaremill/elasticmq-native
+
       - name: Checkout Repo
         uses: actions/checkout@v3
+
       - name: Set up Python
         uses: actions/setup-python@v2
         with:
           python-version: ${{ matrix.python-version }}
+
+      - name: Install poetry
+        run: |
+          python -m pip install poetry
+          poetry install
```

Comment:
This doesn't install `poetry`, but the project dependencies. (i.e., duplicate of what's just below.)